sklearn/datasets/samples_generator.py

```diff
@@ -48,9 +48,13 @@ def make_classification(n_samples=100, n_features=20, n_informative=2,
     class. It introduces interdependence between these features and adds
     various types of further noise to the data.
 
-    Prior to shuffling, `X` stacks a number of these primary "informative"
-    features, "redundant" linear combinations of these, "repeated" duplicates
-    of sampled features, and arbitrary noise for and remaining features.
+    Without shuffling, `X` horizontally stacks features in the following
```

Comment:
Single backticks do nothing with our documentation config. You need double backticks.

Reply:
Thanks. Good to know. Can I go ahead and change single backticks to double backticks for variable names throughout the rest of the docstring as well?

Reply:
I went ahead and changed single backticks to double backticks where appropriate in the rest of the `make_classification` docstring. I can to revert back if this is not what you intended.